openapi/spec.json

```diff
@@ -1177,6 +1183,168 @@
         }
       }
     },
+    "/devices/{organizationId}/{deviceId}/routes/{routeId}/activate": {
+      "post": {
+        "tags": [
+          "Devices"
+        ],
+        "summary": "Activate a device route",
+        "description": "Activate a route for one of your organization's devices. The device will be provisioned with the provider if necessary.",
+        "responses": {
+          "204": {
+            "$ref": "#/components/responses/204"
+          },
+          "401": {
+            "$ref": "#/components/responses/401"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          },
+          "429": {
+            "$ref": "#/components/responses/429"
+          },
+          "500": {
+            "$ref": "#/components/responses/500"
+          }
+        }
+      }
+    },
+    "/devices/{organizationId}/{deviceId}/routes/{routeId}/deactivate": {
```

Comment:
What's the returned status if we deactivate an already activated route?

Reply:
"404 Not Found"

Reply:
![image](https://user-images.githubusercontent.com/19814268/51763225-5141d280-20d2-11e9-9c9b-8f1b2cae9721.png)
I will add this to the doc